src/cli/init/index.ts

```diff
@@ -24,20 +24,22 @@ import { createDir, sanitizeName, write } from "./utils";
 import { confirm, PromptOptions, walkthrough } from "./walkthrough";
 import { ErrorList } from "../../lib/errors";
 import { UUID_LENGTH_LIMIT } from "./enums";
+import { Option } from "commander";
 
 export default function (program: RootCmd): void {
   let response = {} as PromptOptions;
   let pkgOverride = "";
   program
     .command("init")
     .description("Initialize a new Pepr Module")
-    .option("--confirm", "Skip verification prompt when creating a new module.")
-    .option("--description <string>", "Explain the purpose of the new module.")
-    .option("--name <string>", "Set the name of the new module.")
-    .option("--skip-post-init", "Skip npm install, git init, and VSCode launch.")
-    .option(`--errorBehavior <${ErrorList.join("|")}>`, "Set an errorBehavior.")
+    .option("-d, --description <string>", "Explain the purpose of the new module.")
+    .addOption(
+      new Option("-e, --error-behavior <behavior>", "Set an error behavior.").choices(ErrorList),
+    )
+    .option("-n, --name <string>", "Set the name of the new module.")
+    .option("-s, --skip-post-init", "Skip npm install, git init, and VSCode launch.")
     .option(
-      "--uuid [string]",
+      "-u, --uuid <string>",
```

Comment:
This does not require a string, it is optional and without the string it will show you the UUID incase you do not know it